src/msg/async/rdma/RDMAStack.h

```diff
@@ -218,6 +218,7 @@ class RDMAConnectedSocketImpl : public ConnectedSocketImpl {
   virtual void shutdown() override;
   virtual void close() override;
   virtual int fd() const override { return notify_fd; }
+  virtual int set_priority(int sd, int prio, int domain) {return 0;}
```

Comment:
Not sure about DPDK, but why are you stubbing out RDMA?  The existing messenger-wide setting seems to cover RDMA, see `RDMAConnectedSocketImpl::try_connect()` and `RDMAServerSocketImpl::accept()`.

Reply:
OK,  forget it due to first coding on outdated master

Reply:
1. add modifycation for RDMAConnectedSocketImpl
2.  DPDK seems not support  all socket configs yet, so choose to implement always support.